cdap-docs/examples-manual/source/examples/sport-results.rst

```diff
@@ -0,0 +1,212 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: Cask Data Application Platform WordCount Application
+    :copyright: Copyright © 2014-2015 Cask Data, Inc.
+
+.. _examples-fileset:
+
+=============
+Sport Results
+=============
+
+A Cask Data Application Platform (CDAP) Example demonstrating partitioned file sets through sport results analytics.
+
+Overview
+========
+
+This Application demonstrates the use of the PartitionedFileSet dataset:
+
+   - Game results are stored in the PartitionedFileSet ``results``. It is partitioned by league and season,
+     and each partition is a CSV (comma-separated values) file containing the results in one league for a season;
+     for example, the 2014 season of the NFL (National Football League).
+   - Results are uploaded into the file set using the ``UploadService``.
+   - The ``ScoreCounter`` MapReduce program reads game results for a given league and
+     aggregates total counts such as games won and lost, or points scored and conceded, across all
+     seasons, and writes these totals to the partitioned file set ``totals`` that is partitioned by league.
+   - Both the original game results and the aggregated totals can be explored using ad-hoc SQL queries.
+
+Let's look at some of these components, and then run the Application and see the results.
+
+The SportResults Application
+----------------------------
+
+As in the other :ref:`examples,<examples-index>` the components
+of the Application are tied together by the class ``SportResults``:
+
+.. literalinclude:: /../../../cdap-examples/SportResults/src/main/java/co/cask/cdap/examples/sportresults/SportResults.java
+    :language: java
+    :lines: 29-
+
+The ``configure()`` method creates the two PartitionedFileSet datasets used in this example.
+
+   - Both datasets use CSV as the format: For MapReduce, they use the TextInputFormat and TextOutputFormat
+     with "," as the field separator, and for Explore they use the ``csv`` format.
+   - The first dataset (``results``) is partitioned by league and season. Each record represents
+     a single game with a date, a winning and a losing team, and the winner's and the loser's points, for example::
+
+       2011/9/5,Dallas Cowboys,New York Giants,24,17
+       2011/9/9,Philadelphia Eagles,Cleveland Browns,17,16
+       2011/9/9,New England Patriots,Tennessee Titans,34,13
+
+     We have included some sample data in the ``resources`` directory.
+
+   - The ``totals`` dataset stores aggregates across all seasons and therefore has the league as its single
+     partitioning field. Each record represents the total number of games won and lost, and the total number
+     of points scored and conceded, by a team.
```

Comment:
Each record has, for an individual team, the total number of games won and lost and the total number
of points scored and conceded.